client/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "light-client"
+version = "0.8.0"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/lightprotocol/light-protocol"
+description = "Client library for Light Protocol"
+
+[dependencies]
+# Solana
+solana-banks-client = { workspace = true }
+solana-client = { workspace = true }
+solana-sdk = { workspace = true }
+solana-transaction-status = { workspace = true }
+
+# Anchor
+anchor-lang = { workspace = true }
```

Comment:
can we get rid of the anchor dep here?

Reply:
Yes, we can. We were using only borsh and solana-program re-imported modules. I removed it now.